modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration-suma.adoc

```diff
@@ -13,12 +13,12 @@ This section describes migrating from Proxy 4.3 using the [command]``mgrpxy`` to
 
 [IMPORTANT]
 ====
-An in-place migration from {productname} 4.3 to 5.0 is unsupported due to the HostOS change from {sles} 15 SP4 to {sle-micro} {microversion}.
+An in-place migration from {productname} 4.3 to 5.0 is unsupported because the host operating system changed from {sles} 15 SP4 to {sle-micro} {microversion}.
```

Comment:
```suggestion
An in-place migration from {productname} 4.3 to 5.0 is unsupported because the host operating system has changed from {sles} 15 SP4 to {sle-micro} {microversion}.
```